mathics_scanner/data/operators.yml

```diff
@@ -7839,7 +7866,7 @@ Xnor:
 
 Xor:
   actual-precedence: 280
-  precedence: 215
+  precedence: 220
```

Comment:
In WL this is 215

Reply:
I will make note of this for the future in a comment FIXME. 

The change to Mathics core which is coming up will be a bit large and probably painful for me.  I will probably have to make several passes to get everything tidy. 

But initially, I am going to be conservative and go with existing Mathics code. After things pass I can go back and fix up the list of discrepancies that were found.